app/scripts/ui.js

```diff
@@ -43,7 +43,9 @@ async function start () {
   // provide app state to append to error logs
   function getState () {
     // get app state
-    const state = window.getCleanAppState()
+    const state = window.getCleanAppState
```

Comment:
I see what you're doing here, but I am left wondering why this is even on `window`?

If we're going to check the prop here maybe we should use `window.hasOwnProperty('getCleanAppState')`?

Reply:
What cases would this handle?

Reply:
https://stackoverflow.com/questions/34128945/javascript-hasownproperty-vs-dot-syntax

for this it's probably fine? I was just thinking from a convention point of view

Reply:
I think for this it's fine, I don't quite see what benefit this change would bring